src/main/resources/application.yml

```diff
@@ -168,3 +168,6 @@ kafka:
       topic-pattern: (${environment}\.)(.*\.)inventory\.holdings-record
       concurrency: ${KAFKA_EVENTS_CONCURRENCY:2}
       data-type: org.folio.innreach.dto.Holding
+  backoff:
+    interval: 30000
+    max_failure: 5
```

Comment:
Thinking ahead to how we're going to want to do the same thing for the instances of consumers for _ongoing contributions_. If we set these globally in the app yaml are they going to apply to _every instance_ of our use of the  `ConcurrentMessageListenerContainer`? If so that would be bad.

I think the retry interval and the max_failure is probably completely different for initial contrib vs ongoing.

Reply:
How has this been resolved? I see that you have increased the interval and the max failure, but how do the new values you have chosen address my concerns?